File: Next_Layer_Boost_App-main/app/src/main/java/com/sis/clighteningboost/Utills/DateUtils.java
package com.sis.clighteningboost.Utills;

import java.text.SimpleDateFormat;
import java.util.Date;

public class DateUtils {
    public static String getCurrentDate(){
        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
        String date = format.format(new Date());
        return date;
    }
}

File: Next_Layer_Boost_App-main/app/src/main/java/com/sis/clighteningboost/utils/DateUtils.kt
package com.sis.clighteningboost.utils

import java.text.SimpleDateFormat
import java.util.*

object DateUtils {
    @JvmStatic
    val currentDate: String
        get() {
            val format =
                SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
            return format.format(Date())
        }
}